.mvn/modernizer/violations.xml

```diff
@@ -277,4 +277,10 @@
         <version>1.8</version>
         <comment>Use io.airlift.slice.SizeOf.instanceSize</comment>
     </violation>
+
+    <violation>
+        <name>java/util/Locale.ROOT:Ljava/util/Locale;</name>
```

Comment:
This seems overly broad. There might be legitimate uses for that locale. In fact, it's possible that the right locale for many of the uppercase/lowercase operations should be Locale.ROOT, but we'd need to look into it further.